config/path.js

```diff
@@ -9,56 +9,6 @@
 
```

Comment:
it could be written as json, however, it then requires an additional action to parse the data before we can loop over the data.